### PR TITLE
Add missing `tf.keras` loss functions aliases

### LINKED ISSUE #9699: [Keras model import] Wrong loss function configuration import mechanism for `tf.keras`
#### Issue Description

I exported a Keras model like this:

```
import tensorflow as tf
model = tf.keras.Sequential([
    tf.keras.layers.Flatten(input_shape=(28, 28)),
    tf.keras.layers.Dense(128, activation='relu'),
    tf.keras.layers.Dense(10)
])
model.compile(optimizer='adam',
              loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
              metrics=['accuracy'])

...

model.save('my_model.h5', overwrite=True, include_optimizer=True)
```

I tried to import a HDF5 Keras model file:

`MultiLayerNetwork model = KerasModelImport.importKerasSequentialModelAndWeights("<directory>/my_model.h5");`

and got this exception:

```
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unknown Keras loss function sparsecategoricalcrossentropy. Please file an issue at https://github.com/eclipse/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLossUtils.mapLossFunction(KerasLossUtils.java:99)
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasLoss.<init>(KerasLoss.java:83)
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasLoss.<init>(KerasLoss.java:61)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.importTrainingConfiguration(KerasModel.java:413)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:139)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:55)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildSequential(KerasModelBuilder.java:326)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:218)
```

I opened the HDF5 model file with HDFView to investigate and what I got for the `training_config` is (prettified):

```
{
  "loss":{
    "class_name":"SparseCategoricalCrossentropy",
    "config":{
      "reduction":"auto",
      "name":"sparse_categorical_crossentropy",
      "from_logits":true
    }
  },
  "metrics":[
    [
      {
        "class_name":"MeanMetricWrapper",
        "config":{
          "name":"accuracy",
          "dtype":"float32",
          "fn":"sparse_categorical_accuracy"
        }
      }
    ]
  ],
  "weighted_metrics":null,
  "loss_weights":null,
  "optimizer_config":{
    "class_name":"Adam",
    "config":{
      "name":"Adam",
      "learning_rate":0.0010000000474974513,
      "decay":0.0,
      "beta_1":0.8999999761581421,
      "beta_2":0.9990000128746033,
      "epsilon":1e-07,
      "amsgrad":false
    }
  }
}
```

So I think there is a mismatch when importing loss function configuration here. I checked the `importTrainingConfiguration` method code, and the method is reading the `class_name` attribute instead of the `name` of `config` attribute. I wonder if previous version of `Keras` serializes the model differently.

#### Version Information

- Deeplearning4j version: `1.0.0-M2`
- `tf.keras.__version__`: `2.9.0`

### PR DESCRIPTION
**Pull request by thangnbn**
Signed-off-by: Thang Bui <buinnthang@gmail.com>

## What changes were proposed in this pull request?

Fix #9699 - This PR adds aliases of those `tf.keras` loss functions that PR #9702 missed.

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [X] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [X] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.

### CODE REVIEW

**Review by agibsonccc — APPROVED** [review]
Ah good catch! Thanks a lot for following the fix.
